### PR TITLE
Update permissions for private locations

### PR DESCRIPTION
**Pull request by paulb-elastic**
Update required permissions as a result of https://github.com/elastic/kibana/pull/159378

### CODE REVIEW

File: docs/en/observability/synthetics-role-write.asciidoc [hunk 1]
```diff
@@ -83,8 +83,8 @@ Additional privileges will depend on the factors below.
 [discrete]
 == If using Private Locations
 
-If the user should be able to create and update monitors hosted on
-<<synthetics-private-location,Private Locations>>, add the following privileges:
+No additional permissions are needed if the user should be able to create and update monitors hosted on
+<<synthetics-private-location,Private Locations>>. However when the agent policy is initially created within {fleet}, the following privileges are required:
```
Comment (colleenmcginnis) [inline docs/en/observability/synthetics-role-write.asciidoc:87]: This is confusing to me. Is this correct? 

>The user who initially sets up a new Private Location needs additional privileges. Users who create or update monitors hosted on that Private Location do not need any additional permissions.
>
>The user who is setting up a new Private Location will need the following privileges when creating the agent policy in {fleet}:

Comment (paulb-elastic) [reply docs/en/observability/synthetics-role-write.asciidoc:87]: @colleenmcginnis kind of, but there is a nuance, but don’t know if it complicates the message.  To create the Agent Policy (under Fleet), the additional permissions are needed.  However, once it exists, the user doesn’t need the Fleet permissions to add the Private Location (under Synthetics > Settings).

Comment (shahzad31) [reply docs/en/observability/synthetics-role-write.asciidoc:87]: @paulb-elastic 
we only need fleet all permissions to create an agent policy, we can also remove 

```
| {kibana-ref}/kibana-privileges.html[Kibana]
| `Integrations`: `All`
| Access to Integrations in {kib}.
```

section

Comment (paulb-elastic) [reply docs/en/observability/synthetics-role-write.asciidoc:87]: @colleenmcginnis + @shahzad31, I've updated with both your feedback

**Comment by paulb-elastic** [comment]
@colleenmcginnis / @shahzad31 can you take another look please, thanks

**Review by shahzad31 — APPROVED** [review]
LGTM !!

File: docs/en/observability/synthetics-role-write.asciidoc [hunk 2]
```diff
@@ -83,8 +83,12 @@ Additional privileges will depend on the factors below.
...
+
+The user who initially sets up a new Private Location needs additional privileges. Users who create or update monitors hosted on that Private Location do not need any additional permissions.
+
+The user who is setting up a new Private Location will need the following privileges when creating the agent policy in {fleet}:
```
Comment (colleenmcginnis) [inline docs/en/observability/synthetics-role-write.asciidoc:91]: Did you mean to delete the first paragraph? It currently [looks like this](https://observability-docs_3008.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-role-write.html#_if_using_private_locations): 

Screenshot 2023-06-22 at 12 50 18 PM

```suggestion
The user who initially sets up a new Private Location needs additional privileges. Users who create or update monitors hosted on that Private Location do not need any additional permissions.

The user who is setting up a new Private Location will need the following privileges when creating the agent policy in {fleet}:
```

**Comment by paulb-elastic** [comment]
Thanks @colleenmcginnis, not quite sure what's happened there.  I've updated as per your suggestion, ready for your review.

**Review by colleenmcginnis — APPROVED** [review]
✨

File: docs/en/observability/synthetics-role-write.asciidoc [hunk 3]
```diff
@@ -94,10 +95,6 @@ If the user should be able to create and update monitors hosted on
 | `Fleet`: `All`
 | Access to Fleet in {kib}.
 
-| {kibana-ref}/kibana-privileges.html[Kibana]
-| `Integrations`: `All`
-| Access to Integrations in {kib}.
-
```
Comment (shahzad31) [inline docs/en/observability/synthetics-role-write.asciidoc:100]: we will need to revert this part. i just test on cloud for POST FF and user needs at least read permission as well to access fleet UI to create an agent policy.

Comment (paulb-elastic) [reply docs/en/observability/synthetics-role-write.asciidoc:100]: https://github.com/elastic/observability-docs/pull/3044 raised

**Pull request merged**